dpks/differential_testing.py

```diff
@@ -39,16 +39,20 @@ def __init__(
 
         elif level == "peptide":
             self.level = "PeptideSequence"
+        else:
+            self.level = level
 
-    def test(self, quantitative_data: QuantMatrix) -> QuantMatrix:
+    def test(self, quant_matrix: QuantMatrix) -> QuantMatrix:
```

Comment:
It's called ```quantitative_data``` everywhere else in the package, if we are going to change this, make sure any changes are propagated to be consistent. We may want to change the ```quant_matrix``` module name, as these may now clash.

Was there a particular reason for this change?

Reply:
The reason I wanted to change it was because I got really confused about QuantMatrix being called quantitative_data since it has a property called quantitative_data. So quantitative_data.quantitative_data was called everywhere which was pretty confusing. quant_matrix.quantitative_data makes more sense to me.

Reply:
I can't find where else it's quantitative_data. Both DE and quant is changed.